example/lib/ai_analysis_ex.dart

```diff
@@ -72,9 +72,11 @@ class _CameraPageState extends State<CameraPage>
         children: <Widget>[
           Positioned.fill(
             child: CameraAwesomeBuilder.awesome(
-              initialCaptureMode: CaptureModes.PHOTO,
-              picturePathBuilder: (captureMode) => _path(captureMode),
-              videoPathBuilder: (captureMode) => _path(captureMode),
+              awesomeFileSaver: AwesomeFileSaver.imageAndVideo(
+                imagePathBuilder: () => _path(CaptureModes.PHOTO),
+                videoPathBuilder: () => _path(CaptureModes.VIDEO),
+                initialCaptureMode: CaptureModes.PHOTO,
```

Comment:
c'est bizarre d'avoir le initialCaptureMode dans le paramètre awesomeFilesaver ?
On pourrait renommer le paramètre awesomeFileSaver en pathBuilders ?

Reply:
Renamed into SaveConfig

Reply:
Leaving this style recommendation here for the enums: https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names